src/argilla/tasks/users/create_default.py

```diff
@@ -37,7 +37,7 @@ def create_default(
             User(
                 first_name="",
                 username=DEFAULT_USERNAME,
-                role=UserRole.admin,
+                role=UserRole.owner.value,
```

Comment:
The `User` model uses  `UserRole` for the `role` attribute. So we can use the enum instance
```suggestion
                role=UserRole.owner,
```